core/extensions.js

```diff
@@ -243,10 +232,10 @@ Blockly.Extensions.checkMutatorDialog_ = function(object, errorPrefix) {
  */
 Blockly.Extensions.checkBlockHasMutatorProperties_ = function(errorPrefix,
```

Comment:
This function expects (errorPrefix, block) but is called with (name, block, errorPrefix).  How does this work?